components/metal/IntroSection.vue

```diff
@@ -54,6 +64,7 @@ export default class extends Vue {
 
   &__title {
     @include type-style('expressive-heading-05');
+    max-width: 500px;
```

Comment:
For the sake of sticking with Carbon mini-units (8px), can we use a value here that's a multiple of 8?

`512px`, for example, or `32rem`?

Reply:
I realize that we have values around the code base that don't stick to this mini-unit principle all the time, but it'd be nice to use it when we can.


thoughts? 

or does this not make sense to do?

Reply:
If a value doesn't stick to the mini-unit principle is because it was done before we started using the carbon design system. Now we try to use them